azext_iot/central/commands_device.py

```diff
@@ -5,31 +5,69 @@
 # --------------------------------------------------------------------------------------------
 # Dev note - think of this as a controller
 
+from azext_iot.central.common import EDGE_ONLY
```

Comment:
To avoid confusion with edge_only in list_devices, you should rename this to EDGE_ONLY_FILTER

Reply:
+1